nemo_skills/inference/generate.py

```diff
@@ -353,17 +349,6 @@ def generate(cfg: GenerateSolutionsConfig):
             f"Number of samples in the chunk: {len(data)}"
         )
 
```

Comment:
Were dropping offset support?

Reply:
did you use it? I just think it was never used, so decided to remove, but if you're using it, we can keep. It's just getting hard to use properly with all the extra settings we add